src/nvim/options.lua

```diff
@@ -10257,17 +10257,17 @@ local options = {
     },
     {
       defaults = { if_true = '' },
-      values = { '', 'double', 'single', 'shadow', 'rounded', 'solid', 'bold', 'none' },
+      values = { '', 'bold', 'double', 'none', 'rounded', 'shadow', 'single', 'solid' },
```

Comment:
enum of `opt_winborder_values` also need update https://github.com/neovim/neovim/blob/8a40213eb38dcdcb4c59dda2fe088f5919e903f2/src/nvim/api/win_config.c#L940

Reply:
oh yikes. i'll drop this change for now.